src/cljs/main/broadfcui/page/method_repo/method/exporter.cljs

```diff
@@ -209,4 +210,9 @@
                   (if success?
                     ((:on-export props) workspace-id (ws-common/config->id config))
                     (utils/multi-swap! state (assoc :server-error (get-parsed-response false))
-                                             (dissoc :banner))))}))})
+                                             (dissoc :banner)))
+                  (endpoints/send-metrics-event
+                   "workflow:export:firecloud"
+                   {:success success?
```

Comment:
Should we only send this event on success? I like the idea of capturing failures somehow, though we might want to capture more/different details when recording failures... which might require just enough more thought to make it slightly out-of-scope.

Reply:
We capture failures for workspace sharing, so I figured we should here, too.